app/src/main/res/layout/fragment_folder.xml

```diff
@@ -221,38 +221,6 @@
                     android:layout_marginStart="75dp"
                     android:textAppearance="@style/TextAppearance.AppCompat.Caption" />
 
-            </LinearLayout>
-
-            <!-- variableSizeBlocks -->
-            <android.support.v7.widget.SwitchCompat
-                android:id="@+id/variableSizeBlocks"
-                style="@style/Widget.Syncthing.TextView.Label.Details"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:checked="false"
-                android:drawableLeft="@drawable/ic_developer_board_black_24dp"
-                android:drawableStart="@drawable/ic_developer_board_black_24dp"
-                android:text="@string/folder_variable_size_blocks_caption" />
-
-            <TextView
-                android:id="@+id/variableSizeBlocksDescription"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="75dp"
-                android:layout_marginStart="75dp"
-                android:layout_marginTop="-20dp"
-                android:textAppearance="@style/TextAppearance.AppCompat.Caption"
-                android:text="@string/folder_variable_size_blocks_description"
-                android:focusable="false"/>
-
-            <LinearLayout
-                android:id="@+id/editIgnoresContainer"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:background="?selectableItemBackground"
-                android:orientation="vertical"
-                android:gravity="center_vertical">
-
                 <TextView
```

Comment:
Continuing from https://github.com/syncthing/syncthing-android/pull/1344#issuecomment-500565690:  
What made me think this belongs together with the remove paragraph above is that this part is indented compared to the removed `LinearLayout`, i.e. before it looked like it was a child of the editIgnoresContainer, now it's a child of something much further up. If that has no particular meaning or is fine, then just ignore this.

Reply:
I have no idea, I looked for the string, it's not there. I don't think any of this stuff is parent-childish, it's just a list of stuff.